swarm/network/stream/snapshot_sync_test.go

```diff
@@ -67,56 +55,14 @@ type synctestConfig struct {
 	addrToIdMap      map[string]discover.NodeID
 }
 
-func init() {
-	rand.Seed(time.Now().Unix())
-}
-
-//common_test needs to initialize the test in a init() func
-//in order for adapters to register the NewStreamerService;
-//this service is dependent on some global variables
-//we thus need to initialize first as init() as well.
-func initSyncTest() {
-	//assign the toAddr func so NewStreamerService can build the addr
-	toAddr = func(id discover.NodeID) *network.BzzAddr {
-		addr := network.NewAddrFromNodeID(id)
-		return addr
-	}
-	//global func to create local store
-	if *useMockStore {
-		createStoreFunc = createMockStore
-	} else {
-		createStoreFunc = createTestLocalStorageForId
-	}
-	//local stores
-	stores = make(map[discover.NodeID]storage.ChunkStore)
-	//data directories for each node and store
-	datadirs = make(map[discover.NodeID]string)
-	//deliveries for each node
-	deliveries = make(map[discover.NodeID]*Delivery)
-	//registries, map of discover.NodeID to its streamer
-	registries = make(map[discover.NodeID]*TestRegistry)
-	//not needed for this test but required from common_test for NewStreamService
-	waitPeerErrC = make(chan error)
-	//also not needed for this test but required for NewStreamService
-	peerCount = func(id discover.NodeID) int {
-		if ids[0] == id || ids[len(ids)-1] == id {
-			return 1
-		}
-		return 2
-	}
-	if *useMockStore {
-		createGlobalStore()
-	}
-}
-
 //This test is a syncing test for nodes.
 //One node is randomly selected to be the pivot node.
 //A configurable number of chunks and nodes can be
 //provided to the test, the number of chunks is uploaded
 //to the pivot node, and we check that nodes get the chunks
 //they are expected to store based on the syncing protocol.
 //Number of chunks and nodes can be provided via commandline too.
-func TestSyncing(t *testing.T) {
+func TestSyncingViaRegistry(t *testing.T) {
```

Comment:
these two test could be named better.
one is using direct subscribe calls, the other the global sync method